echopype/calibrate/api.py

```diff
@@ -34,11 +58,159 @@ def _compute_cal(
         return cal_obj.compute_Sp(waveform_mode=waveform_mode, encode_mode=encode_mode)
 
 
-def compute_Sv(echodata: EchoData, **kwargs):
-    """Compute volume backscattering strength (Sv) from raw data."""
+def compute_Sv(echodata: EchoData, **kwargs) -> xr.Dataset:
+    """
+    Compute volume backscattering strength (Sv) from raw data.
+
+    The calibration routine varies depending on the sonar type.
+    Currently this operation is supported for the following ``sonar_model``:
+    EK60, AZFP, EK80 (see Notes below for detail).
+
+    Parameters
+    ----------
+    echodata : EchoData
+        An `EchoData` object created by using `open_raw` or `open_converted`
+
+    env_params : dict
```

Comment:
env_params and cal_params are also optional, so for consistency with your use of the qualifier "optional" in the other parameters, I'd add that qualifier to these two. More generally, we're not currently consistent across the board (across all echopype functions/classes) in having a parameter "optional" qualifier string. But that's a broader topic.

Reply:
> More generally, we're not currently consistent across the board (across all echopype functions/classes) in having a parameter "optional" qualifier string. 

Could you add this to #478 ?

Reply:
the "optional" qualifier is added to both env_params and cal_params docstrings